### ISSUE TITLE
Align .ss with registry.nic.ss listing and confirm with Registry Admin

### ISSUE THREAD

**Issue by dnsguru** [publicsuffix/list#983]
I have an email out to an administrator of .ss that had addressed an expired DNSSEC issue in the .ss zone, as I caught some disparity between their documented zones and those in the PSL.

Looks like we need to add edu.ss, me.ss, and sch.ss and confirm ss will accept second level registrations.

registry.nic.ss says (on [https://registry.nic.ss/login.jsp](https://registry.nic.ss/login.jsp)):
```
com.ss
net.ss
org.ss
biz.ss
**edu.ss**
**sch.ss** 
gov.ss
**me.ss**

```
PSL Currently says:
```
// ss : https://registry.nic.ss/
// Submitted by registry <technical@nic.ss>
ss
biz.ss
com.ss
edu.ss
gov.ss
net.ss
org.ss
```

**Comment by dnsguru** [comment]
I have encouraged the person I reached at .SS to join github and engage in the discussion to confirm we have this correct, and to add the appropriate _PSL TXT records to the zones accordingly
ss
com.ss
net.ss
org.ss
biz.ss
edu.ss
sch.ss
gov.ss
me.ss

**Comment by dnsguru** [comment]
This would revise #882

**Comment by ahmedlandi** [comment]
Hallo Frakes,

Thank you for reaching out. DNSSEC issue has since been resolved.

Also you my add edu.ss, sch.ss and me.ss to PSL.

Please let me know if there is anything else i can assist with.

**Comment by dnsguru** [comment]
This was all addressed in merged item #882 - closing
